packages/legend-query-builder/src/components/fetch-structure/QueryBuilderResultModifierPanel.tsx

```diff
@@ -58,20 +70,26 @@ const ColumnSortEditor = observer(
       label: sortState.columnState.columnName,
       value: sortState,
     };
+    const sortType = sortState.sortType;
```

Comment:
just personal preference, since we onlu use it once, maybe we could remove this variable